lib/workers/repository/process/write.ts

```diff
@@ -39,11 +63,38 @@ export async function writeUpdates(
       meta['baseBranch'] = baseBranch;
     }
     addMeta(meta);
-    const branchExisted = branchExists(branch.branchName);
+    const branchExisted = branchExists(branchName);
+    let branchCache = {} as BranchCache;
+    if (branchExisted && config.repositoryCache === 'enabled') {
```

Comment:
```suggestion
    if (branchExisted) {
```

I don't think we need this check, as we have a null cache when not enabled, so `getCache()` always returns empty object 🤔